core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -48,6 +48,7 @@ object SystemResources {
   def apply(cores: Option[Cores] = None, totalMemory: Option[Memory] = None) : SystemResources = {
     val heapSize = Resource.heapSize
     val system   = totalMemory match {
+      case Some(memory) if heapSize >= memory => throw new IllegalArgumentException(s"Heap size ${heapSize.prettyString} (-Xmx) must greater than or equal to the total memory of ${memory.prettyString}.")
```

Comment:
The message here seems backwards.  You're saying if the `heapSize >= memory` it's bad, but the message says it must be greater than the total memory.  Presumably you mean less...